app/components/ak-button/neutral-outlined/index.hbs

```diff
@@ -0,0 +1,13 @@
+<button
+  type="button"
+  disabled={{@isDisabled}}
+  local-class="neutral-outlined"
+  ...attributes
+  data-test-btn-neutral-outlined
+  {{on 'click' @onClick}}
+>
+  {{#if @icon}}
```

Comment:
this should not exist here...it should be yielded along with text

Reply:
It was in thought process but later on decided to wrap the contents within component itself. Moved now